pkg/cmd/taskrun/delete.go

```diff
@@ -17,14 +17,15 @@ package taskrun
 import (
 	"errors"
 	"fmt"
+	trsort "github.com/tektoncd/cli/pkg/taskrun/sort"
```

Comment:
The linter is complaining about this import. Try running `gofmt -w` on this file to resolve the way your imports are are formatted.